gallery/css/custom.css

```diff
@@ -0,0 +1,16 @@
+.view {
+  max-height: 180px;
+  object-fit: cover;
+  object-position: center;
+  width: 300px;
+}
+
+.logo {
+  max-height: 45px;
+  max-width: 80px;
```

Comment:
@baamaboi don't style anything with px unit. This won't make our page responsive. Go for rem, vw, vh etc.

Reply:
OK, will change that